asv_bench/benchmarks/pandas_vb_common.py

```diff
@@ -1,27 +1,29 @@
 import os
-from pandas import *
-import pandas as pd
-from numpy.random import randn
-from numpy.random import randint
-import pandas.util.testing as tm
-import random
-import numpy as np
-import threading
 from importlib import import_module
 
+import numpy as np
 try:
-    from pandas.compat import range
+    from pandas import Panel
 except ImportError:
-    pass
+    from pandas import WidePanel as Panel  # noqa
+
+# Compatibility import for lib
+for imp in ['pandas._libs.lib', 'pandas.lib', 'pandas_tseries']:
```

Comment:
this last is wrong ``pandas_tseries``, where did this come from?

Reply:
Not sure; `pandas_tseries` was present here when I started the clean-up. I can remove this

Reply:
ok let's do, otherwise lgtm.